seahorn/jobs2/array_list_clear2/aws_array_list_clear_harness2.c

```diff
@@ -19,18 +19,20 @@ int main() {
     assume(aws_array_list_is_valid(&list));
 
     /* save current state of the data structure */
-    struct aws_array_list old = list;
+    sea_tracking_on();
+    size_t old_current_size = list.current_size;
+    size_t old_item_size = list.item_size;
 
     /* perform operation under verification */
     aws_array_list_clear(&list);
 
-    /* assertions */
+   /* assertions */
     sassert(aws_array_list_is_valid(&list));
+    sassert(!sea_is_modified((char *)list.alloc));
```

Comment:
This check is different from `sassert(list.alloc == old.alloc)`. It is much stronger and it is what we want.

Reply:
So this doesn't need any changes?

Reply:
Yes LGTM